README.md

```diff
@@ -96,3 +97,4 @@ License
 -------
 
 This code is distributed under the MIT license, see the [LICENSE](https://github.com/meltano/meltano/blob/main/LICENSE) file.
+<p align="right">(<a href="#top">🔼 Back to top</a>)</p>
```

Comment:
Thanks for this contribution @PrajwalGraj! I think that we should make it clear that this is not part of the above section by adding a horizontal bar, center the element, and remove the parentheses. I've implemented those suggestions here:

```suggestion

---

<p align="center"><a href="#top">🔼 Back to top</a></p>
```

Let me know what you think. If this seems good to you, I'll go ahead and approve & merge this.

Reply:
Thank you @WillDaSilva  for your quick response and your willingness to incorporate the changes.Iam Fine and ok with your changes, so you can go ahead and proceed🚀.
Thank you